java/simpleproducer/src/main/java/SimpleProducer.java

```diff
@@ -65,13 +65,13 @@ public static void main(String[] args) throws Exception{
 //                    Integer.toString(i), Integer.toString(i)));
 
         while(true) {
-            producer.send(new ProducerRecord<String, String>(topicName, "abhishek", "abhishek"));
+            producer.send(new ProducerRecord<String, String>(topicName, "abhishek", "(5,[0.1,0.2,0.5])"));
 
             //sleep 1 seconds
             Thread.sleep(1000);
             System.out.print(".");
 
-            producer.send(new ProducerRecord<String, String>(topicName, "goswami", "goswami"));
+            producer.send(new ProducerRecord<String, String>(topicName, "goswami", "(4,[0.1,0.2,0.4])"));
```

Comment:
For now i was using the producer to generate traffic, just to validate things are working fine.  eventually we dont care about this java producer , because the data will be pumped in from flask.